File: SofiForce.Models/Models/SearchModels/ClientRouteSearchModel.cs
using System;
using System.Collections.Generic;

namespace Models
{
    public class ClientRouteSearchModel : BaseSearchModel
    {

		public int? ClientId { get; set; }    
		public int? RouteId { get; set; }
        public int? BranchId { get; set; }
        public int? IsActive { get; set; }

        public int? RouteTypeId { get; set; }


    }
}

File: SofiForce.Web/ClientApp/src/app/core/Models/SearchModels/ClientRouteSearchModel.ts
import { BranchModel } from "../EntityModels/branchModel";
import { RepresentativeModel } from "../EntityModels/representativeModel";
import { StoreModel } from "../EntityModels/storeModel";
import { BaseSearchModel } from "./baseSearchModel";

export interface ClientRouteSearchModel extends BaseSearchModel {
   
    clientId:number;
    clientCode: string;

    branchId: number | null;
    branchCode: string | null;
   
    routeId: number | null;
    routeCode: string | null;

    routeTypeId: number | null;
    isActive: number | null;

   
}